Untitled Game/src/Levels/House.gd

```diff
@@ -35,6 +33,9 @@ onready var _basement_Teleport_Area: Area2D = get_node("LevelBackground/AreaTran
 onready var _foyer_Teleport_Area: Area2D = get_node("LevelBackground/AreaTransitions/Foyer_To_Garage")
 onready var _garage_Teleport_Area: Area2D = get_node("LevelBackground/AreaTransitions/Garage_To_Foyer")
 
+onready var _bathtub_Interact_Area: Area2D = $LevelBackground/Interactions/Kitchen/BathtubInteract
+onready var _textBox: TextBox = $GUI/TextBox
```

Comment:
we should try to abstract this like Exo did with transitions so that we dont have to type in every one